pkg/controller/manager/manager_controller.go

```diff
@@ -359,6 +359,7 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		trustedSecretNames = append(trustedSecretNames, render.ComplianceServerCertSecret)
 	}
 
+	var additionalNameSpaces []string
```

Comment:
nit: Maybe additional uiLayerNamespaces would be more indicative of how this variable is used.